### DISCUSSION TITLE
Numpy

### DISCUSSION THREAD

**Discussion by emiliomastriani** [codewars/codewars.com#3197] [Q&A]
Hello guys, I am doing practice on Python using CodeWars, and it looks interesting. I just have one simple question: suppose I wish to practice Python focusing on some specific library, such as Numpy, or Pandas. Is there a way to select this kind of exercise? Thank you very much. With the best regards. Emilio

**Comment by ghost** [comment]
Unfortunately, at the moment there is no way to select exercises by library
but you can select by math

![image](https://github.com/codewars/codewars.com/assets/162613265/82302050-4052-47dd-bea5-627743617e35)
